### ISSUE TITLE
Can't get armor back on

### ISSUE THREAD

**Issue by wizardcheetah55** [Wynncraft/Issues#3964]
Lvl 75. Considered changing armor. Removed helmet and pants; decided against the change. Now the system won't allow me to put the armor back on - I don't have enough of a category. Going out nearly naked wouldn't be a pretty sight.

Plus, is there a tutorial for after-the-Gavel-upgrade? I need to learn why I am so much slower and weaker (i.e. my Ethereal wand is fast, but takes took many hits to kil a Level 55 Zombie.

Thanks for the help!

**Comment by DarkDemonDragon** [comment]
So when the Gavel update came out all your Skill points were reset because weapons had skill requirements. This way you would be able to reallot your points in a way to equip the armor you want to equip. Also "clean" armor can't be worn anymore. All armor (unless it's normal) MUST be IDed before it can be used.

**Comment by wizardcheetah55** [comment]
Thank you. They were already identified. And now, my wand can't be used, that I've been using for months. A message just came up that said I don't have enough Intelligence points. This has to be a bug - it just happened and I just went from the bank in Troms to identify my armor - that was rejected because it had already been identified. I'm at a loss now.

**Comment by wizardcheetah55** [comment]
Finally, my Autotomized accessory all of a sudden jumped out of its slot. I can't get it to return because I don't have enough agility points. This has to be a bug - all these things don't suddenly happen.

**Comment by DarkDemonDragon** [comment]
Mind posting a screenshot of your skill points and a few of the problematic items?

**Comment by wizardcheetah55** [comment]
![screenshot 2016-02-03 22 57 40](https://cloud.githubusercontent.com/assets/11051478/12808281/68c25648-caca-11e5-987a-82645672eb14.png)
Please note the message in the lower left hand corner. I have two more screenshots for the helmet and pants, that I cannot put back on. I have had them on for months. Thank you for your help.

**Comment by DarkDemonDragon** [comment]
Is your Intelligence 60? Please post a screenshot of your skills (found in your compass)

**Comment by wizardcheetah55** [comment]
Thank you.

![wizardcheetahskillpts](https://cloud.githubusercontent.com/assets/11051478/12823204/98c3252a-cb1f-11e5-803e-d996ade45c03.jpg)

**Comment by Trixter313** [comment]
You just need at least the required amount of skill points to be able to wear certain things and until you can get more, you'll have to use a different weapon that you do meet the requirements for.

**Comment by wizardcheetah55** [comment]
Thank you, but all I'm trying to do is to use the weapon and the armour
that I have had on for months - including after Gavel.

Are you saying that in the middle of a game, the rules change so that what
I have I can no longer have?

The problem this creates is that I'm a Level 75. In order to get more XP I
need to engage with near level mobs; but without my armour and weapon, I
won't be able to compete.

Are you sure there wasn't some kind of glitch in this?

Stan Emert
206-250-9839
Skype: stan.emert

**Comment by Trixter313** [comment]
The game doesn't check to make sure you meet the requirements until you try and remove armor for the first time after the Gavel update so the game didn't realize you weren't supposed to be wearing it until you tried changing your armor. Also, things are getting rebalanced all the time in patches so it is possible that the requirements changed but the first scenario is more likely. It is annoying that you will have to wear slightly worse armor for a bit but if you do some of the new quests you should level up pretty fast. Another option, if you'd like, is you could buy a skill reset scroll and re-apply your skill points so that you can wear everything you have. To do that you must complete the quest Tower of Amnesia and once it is done you will be able to buy the reset scroll for 1 LE. Good luck!

**Comment by wizardcheetah55** [comment]
Thank you.

Stan Emert
206-250-9839
Skype: stan.emert

**Comment by wizardcheetah55** [comment]
I really do appreciate the help, but I don't think I articulated the
problem well.

I am in Troms with *no weapon*, *no spells* and *minimal armor*. *None of
that is my doing*. It's what the developers did, and I'm sure this is an
unintended consequence.

I completed the Tower of Amnesia quest a long time ago, but getting there
will be near impossible because I have no way to fight anything.

Can't I just have my armor and weapon power be restored? That could be
accomplished by reallocating skills points. At least that way, I can get
out.

Absent that, there is no game for me to play.

Stan Emert
206-250-9839
Skype: stan.emert
